feature/quicksettings/build.gradle

```diff
@@ -0,0 +1,68 @@
+plugins {
+    id 'com.android.library'
+    id 'org.jetbrains.kotlin.android'
+    id 'kotlin-kapt'
+}
+
+android {
```

Comment:
nit / more of a todo but we should move these out to a common import
otherwise it's going to be very tedious to update any of these variables like targetSdk and dependency versions